src/convnets/builders/invresmodel.jl

```diff
@@ -9,16 +9,6 @@
                       nclasses::Integer = 1000, kwargs...)
 
 Creates a generic inverted residual model structure with the specified configuration.
-
-# Arguments
-
-    - `scalings`: a tuple of two numbers that specify the width and depth scaling factors.
-    - `block_configs`: This is a `Vector` of `Tuple`s that specifies the configuration of the
-    inverted residual blocks. This can take several forms:
-        + for `dwsep_conv_norm`, or depthwise separable convolutional blocks, the tuple
-        should be of the form `(dwsep_conv_norm, kernel size, output channels, stride,
-        number of repeats, activation function)`. For example, the following configuration
-        is valid: `(dwsep_conv_norm, 3, 64, 1, 1, relu6)`.
```

Comment:
Why was this part of the docstring removed? Otherwise LGTM and we can always do small cleanups to post-merge since this documentation has been wanted for a while :)

Reply:
I wanted to add more detailed docs in a follow-up PR, this was incomplete and thus sometimes misleading 😅